AzureMonitorAgent/agent.py

```diff
@@ -1651,12 +1652,11 @@ def find_vm_distro(operation):
     # platform commands used below aren't available after Python 3.6
```

Comment:
This comment and check seem wrong actually - `platform.dist` and `platform.linux_distribution` were documented in Python 3.7 and removed in Python 3.8.